mlflow/server/js/src/components/modals/TextField.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { FormGroup, ControlLabel, FormControl, HelpBlock } from 'react-bootstrap';
+
+class TextField extends Component {
+  static propTypes = {
+    field: PropTypes.object.isRequired,
+    form: PropTypes.object.isRequired,
+    label: PropTypes.string.isRequired,
+    autoFocus: PropTypes.bool,
+  }
+
+  render() {
+    const field = this.props.field;
```

Comment:
For this to be reusable, we need a way to pass props directly to the `<FormControl>`.

eg `const { form: {touched, errors},  label, field, ...props } = this.props;`

then pass `{...props}` after `{...field}` in the `FormControl`. That way you can customize `onFocus` from the outside.